src/newsroom-context/hooks/useGetTranslationUrl.ts

```diff
@@ -8,6 +8,32 @@ import { getCategoryHasTranslation, getCategoryUrl } from '../../utils';
 import { useCurrentCategory } from './useCurrentCategory';
 import { useCurrentStory } from './useCurrentStory';
 
+// Pulled from SDK types to not leak the `@prezly/sdk` dependency into the client bundle
+enum StoryVisibility {
+    PUBLIC = 'public',
+    EMBARGO = 'embargo',
+    PRIVATE = 'private',
+    CONFIDENTIAL = 'confidential',
+}
+
+function getAllowedTranslationVisibilityValues(story: ExtendedStory): StoryVisibility[] {
+    const { visibility } = story;
+
+    if (visibility === StoryVisibility.EMBARGO) {
+        return [StoryVisibility.EMBARGO, StoryVisibility.PUBLIC];
+    }
+
+    if (visibility === StoryVisibility.PRIVATE) {
+        return [StoryVisibility.PRIVATE, StoryVisibility.PUBLIC];
+    }
+
+    if (visibility === StoryVisibility.CONFIDENTIAL) {
+        return [StoryVisibility.CONFIDENTIAL, StoryVisibility.PRIVATE, StoryVisibility.PUBLIC];
+    }
+
+    return [StoryVisibility.PUBLIC];
```

Comment:
Well duh, I don't know why, but something inside of me told "don't use the switch" this one time 🤔 
I'll update it to switch, it would be much cleaner.